cmd/root/root.go

```diff
@@ -97,7 +97,7 @@ func Execute(cmd *cobra.Command) {
 
 	// Run the command
 	cmd, err := cmd.ExecuteContextC(ctx)
-	if err != nil {
+	if err != nil && err != ErrAlreadyPrinted {
```

Comment:
This should use `errors.Is` so that it also works if (for whatever reason) the error is wrapped.